A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/NDInfo.js

```diff
@@ -0,0 +1,539 @@
+class Jet_MFD_NDInfo extends HTMLElement {
+    constructor() {
+        super(...arguments);
+        this._navMode = Jet_NDCompass_Navigation.NAV;
+        this._navSource = 0;
+        this._showILS = false;
+        this._showET = false;
+        this._dTime = 0;
+        this._chronoValue = 0;
+        this._chronoStarted = false;
+        this._aircraft = Aircraft.A320_NEO;
```

Comment:
Did you write this whole file? or was it a vanilla file copied in then modifications made?

If the latter, please commit the original vanilla file without modifications in one commit, then add your changes in another commit so we can easily see what you changed, thanks!

Reply:
please redo this PR, and commit the the vanilla NDInfo.js in its own commit, so that we can see what changes you made ot it, thanks!

Reply:
Sorry I have a very busy weekend. I have made several improvement to this. So I will redo this PR along with the changes. I also make sure to follow your guidelines to commit the original file first.

Reply:
thanks! I'll close this one in the meantime.